frontend/src/utils/requirementGroupUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { IMajorRequirementGroup } from "../../../common/types";
+
+const reqGroupSortOrder = ["AND", "OR", "RANGE"];
+
+/**
+ *
+ * @param groupMap Mapping of section name to IMajorRequirementGroup validation group
+ */
+export const sortOnValues = (
```

Comment:
We should rename to be more specific about its function:
```suggestion
export const sortRequirementGroupsByConstraint = (
```
Or something else that's more concise, naming is hard 😄